### PR TITLE
Add exported property to ExportAllDeclaration

### LINKED ISSUE #202: Export * as ns from "source" representation
The [proposal](https://spectranaut.github.io/proposal-export-ns-from) to enable the new syntax `export * as ns from "source"` has been merged into EcmaScript standard on master for the 2020 edition.
I don't see any change about it in es202.md, how can i represent it?

### PR DESCRIPTION
**Pull request by mck89**
Closes #202

### CODE REVIEW

File: es2020.md [hunk 1]
```diff
@@ -41,3 +41,16 @@ interface ImportExpression <: Expression {
   node, but it can be an arbitrary expression node.
 
 [ImportExpression]: es2015.md#importdeclaration
+
+# Modules
+
+##  ExportAllDeclaration
+
+```js
+interface ExportAllDeclaration <: ModuleDeclaration {
```
Comment (RReverser) [inline es2020.md]: If it's extending an existing interface, this should say `extend interface`.

Comment (mck89) [reply es2020.md]: I've corrected the code as you said

**Comment by RReverser** [comment]
@loganfsmyth What do you think about this vs `ExportNamespaceDeclaration`? (adding to reviewers as you had some concerns)

**Comment by mysticatea** [comment]
For what it's worth, I think that `ExportAllDeclaration` is proper because it takes all members from `"source"` then exports them. The word "all" affects to source rather than destination, so we can write other export declarations even if `ExportAllDeclaration` existed.

**Comment by mysticatea** [comment]
If `ExportAllDeclaration` means "it exports an ambiguous number of elements (in other words, it needs to load more files to decide the number of exported elements)," then using `ExportNamedDeclaration` is reasonable. But `ModuleSpecifier` has `local` property always, we should fix it.

**Comment by mck89** [comment]
Any update about this?

**Comment by jdalton** [comment]
Just fyi this PR is holding up support in projects like `acorn`.

**Comment by kaicataldo** [comment]
One more friendly ping here.

**Comment by bradzacher** [comment]
Friendly ping here, as this is holding up support for TS3.8 features in typescript-eslint (and thus in prettier as well).

**Pull request merged**